controlplane/pkg/apis/remote/connection/connection.proto

```diff
@@ -69,6 +69,7 @@ message ConnectionEvent {
 
 message MonitorScopeSelector {
     string network_service_manager_name = 1;
+    string destination_network_service_manager_name = 2;
```

Comment:
I'm curious why this is needed here...

Reply:
It is needed by connection monitor on proxy nsmd to connect to network service manager in another domain. It was receving information just about source network service manager before